curvefs/proto/metaserver.proto

```diff
@@ -300,6 +303,21 @@ message MetaServerMetadata {
     // required uint32 checksum = 4;
 }
 
+message AppendS3ChunkInfoRequest {
+    required uint32 poolId = 1;
+    required uint32 copysetId = 2;
+    required uint32 partitionId = 3;
+    required uint32 fsId = 4;
+    required uint64 inodeId = 5;
+    map<uint64, S3ChunkInfoList> s3ChunkInfoAdd = 6;
```

Comment:
Will s3ChunkInfoAdd and s3ChunkInfoRemove exist at the same time？If not, required AppendType{Add, Remove} to show the difference.

Reply:
yes,will exist at the same time